apps/web/components/Modal/Modal.tsx

```diff
@@ -119,35 +120,51 @@ export const Modal: FC<ModalProps> = ({
                 borderRadius="large"
                 className={styles.content}
               >
-                <button onClick={onCloseEvent} className={styles.close}>
-                  <Icon type="close" color="blue400" width="18" height="18" />
-                </button>
+                <Box className={styles.close}>
```

Comment:
Not sure if the modal/dialog has the same design, but we now have one component for that in the UI https://ui.devland.is/?path=/docs/components-dialogprompt--default

Reply:
It's absolutely the same design! Had no idea :D I'm going to switch this modal out